labs/lab3/chapter3.md

```diff
@@ -176,123 +176,102 @@ Save the Dockerfile and exit the editor.
 
 Now we are ready to build the images to test our Dockerfiles.
 
-1. Build each image. When building an image docker requires the path to the directory of the Dockerfile.
+1. Build each image. When building an image podman requires the path to the directory of the Dockerfile.
 
-        $ docker build -t mariadb mariadb/
-        $ docker build -t wordpress wordpress/
+        $ sudo podman build -t mariadb mariadb/
+        $ sudo podman build -t wordpress wordpress/
 
-1. If the build does not return `Successfully built <image_id>` then resolve the issue and build again. Once successful, list the images.
+1. If the build does not succeed then resolve the issue and build again. Once successful, list the images.
 
-        $ docker images
+        $ sudo podman images
 
-1. Create the local directories for persistent storage & set permissions for container runtime.
+1. Create the local directories for persistent storage.
 
-        $ sudo mkdir -p /var/lib/mariadb /var/lib/wp_uploads
-        $ sudo chown 27 /var/lib/mariadb /var/lib/wp_uploads
+        $ mkdir -p ~/workspace/logs ~/workspace/mysql ~/workspace/run ~/workspace/wp_logs
 
-1. Run the database image to confirm connectivity. It takes some time to discover all of the necessary `docker run` options.
+1. Run the wordpress image first. It takes some time to discover all of the necessary `podman run` options.
```

Comment:
is there a reason we changed the order and now start wordpress first? I just ran through this and it didn't seem to work. investigating now

Reply:
ok. this was a copy/paste error on my part. I'd still like to know if there was a technical reason why we changed the order. I think it's more natural for a user/admin to bring up a database before they bring up their app.